metadata-models/src/main/pegasus/com/linkedin/datajob/DataFlowInfo.pdl

```diff
@@ -63,4 +64,15 @@ record DataFlowInfo includes CustomProperties, ExternalReference {
     }
   }
   lastModified: optional TimeStamp
+
+  /**
+   * Environment for this flow
+   */
+  @Searchable = {
+    "fieldType": "TEXT_PARTIAL",
```

Comment:
This should be keyword field, even if this was done else where. There is no need to try to tokenize, stem, etc. This should be exact match or no match especially since they are often abbreviations of real words.

Reply:
+1